tests/unit/TodoList.spec.ts

```diff
@@ -1385,6 +1253,205 @@ describe('TodoList - UI - BCOMP', () => {
   })
 })
 
+describe('TodoList - UI - Incorp Apps', () => {
```

Comment:
Next time someone is in this unit test file, if possible, they should ensure there is a test for all possibly scenarios. I started things off here with several new tests: 2 to test my fix and 2 more to verify that I didn't break another scenario. (These include "negative checking" -- that stuff that isn't supposed to display -- like duplicate subtitles -- doesn't.)

Reply:
Sounds good, definitely agree! Though it may be challenging for just one person to know all of the scenarios without a good time investment. 
Definitely agree its useful to incorporate those negative checks alongside verifying the other items!